cmd/cmdutil/utils.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2024 The argocd-agent Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmdutil
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+func ParseKeepAlivePingInterval(interval string) time.Duration {
```

Comment:
It might make sense to make this function more generic and part of the [internal/env](https://github.com/argoproj-labs/argocd-agent/blob/main/internal/env/env.go) package, so that it streamlines with the rest of the code base.